### PR TITLE
Meta: Update issue/PR templates

### PR DESCRIPTION
**Pull request by fregante**
These explains better what we request and why. Also specify REAL URL because many unnecessarily post incomprehensible fake URLs for no good reason.

- [x] Add templates for meta/discussion and change requests

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -8,8 +8,9 @@ labels: bug
...
 
 1. Make sure the bug is caused by Refined GitHub. Try disabling the extension first.
-2. Include a full URL where the bug appears.
-3. Include a screenshot/gif
+2. Include a REAL URL where the bug appears. If it happens on a private repo, find an equivalent public URL.
+  DO NOT post fake URLs like github.com/<user>/<repo> unless you can't replicate it elsewhere.
+3. Include a screenshot/gif so we can help you better
 
-Issues without a URL/screenshot will be closed
+If we can't see the issue nor can replicate it easily, we can't help.
```
Comment (notlmn) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]:
```suggestion
If we can't see the issue nor replicate it easily, we can't help.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 
-1. LINKED ISSUES:
-   Does this PR close/fix an existing issue? Write something like `Closes #10`
+Thanks for contributing! 🍄 Do not ignore this template, plz.
 
-2. TEST URLS:
-   Add some test URLs
+1. Does this PR close/fix an existing issue? Write something like `Closes #10`
+2. What pages does this PR affect? Include some URLs where you tested the code or where reviewers can test it.
+3. Add a screenshot if your PR makes visual changes
```
Comment (cheap-glitch) [inline .github/pull_request_template.md:7]: Maybe include a few guidelines to make better screenshots? e.g.:
* Clean up the area around the targeted element
* Zoom on the page for small UI changes
* Provide some context only if needed
* Use the "Before" and "After" arrows

I could use those remainders myself :)

Comment (fregante) [reply .github/pull_request_template.md:7]: Yeah I was thinking about guidelines for screenshots in the readme but those either go in the readme (we have guidelines for the description in there) or in contributing

This screenshot is only meant to demonstrate the feature for review, not for the readme. In fact multiple screenshots or gifs are encouraged. See #4046 and #4029

**Comment by fregante** [comment]
OT: @yakov116 let's chill if we get repetitive/unwanted suggestions in my new meta issue (4049, no linking) 🙏

**Review by yakov116 — APPROVED** [review]

**Pull request merged**